client-lite/build/postrm.in.sh

```diff
@@ -11,29 +11,55 @@ run_path=@docs_svc_run_dir_path@
 svc_name=@docs_svc_name@
 svc_config_path=@docs_systemd_cfg_path@
 
-echo "Running post-removal script for $svc_name"
+echo "**** Running post-removal script for $svc_name, args: $1 $2 ****"
 
-systemctl reset-failed $svc_name # Remove ourselves from the failed services list. No-op if never failed earlier.
+do_remove_daemon() {
+    echo "Removing systemd unit file: $svc_config_path"
+    rm $svc_config_path
 
-echo "Removing systemd unit file: $svc_config_path"
-rm $svc_config_path
+    echo "Reloading daemons"
+    systemctl daemon-reload
+}
 
-echo "Reloading daemons"
-systemctl daemon-reload
+do_remove_dirs() {
+    echo "Removing log directory: $log_path"
+    rm -rf $log_path
 
-echo "Removing log directory: $log_path"
-rm -rf $log_path
+    echo "Removing run directory: $run_path"
+    rm -rf $run_path
+}
 
-echo "Removing config directory: $config_path"
-rm -rf $config_path
+do_remove_user_and_group() {
+    echo "Removing user: ${do_user_name}"
+    userdel ${do_user_name}
 
-echo "Removing run directory: $run_path"
-rm -rf $run_path
+    echo "Removing group: ${do_group_name}"
+    groupdel ${do_group_name}
+}
 
-echo "Removing user: ${do_user_name}"
-userdel ${do_user_name}
+do_remove_tasks() {
+    do_remove_daemon
```

Comment:
`|| return` for these

Reply:
Since there is no set -e in this yet, all steps are considered as best-effort.